index.js

```diff
@@ -6,18 +6,23 @@ module.exports = function (opts) {
   require('./lib/array_qsort');
   require('./lib/array_unique');
   require('./lib/fs_scandir');
-  require('./lib/object_watch');
+
   require('./lib/string_format');
   require('./lib/string_hashcode');
 
   // skip prototype pollution of methods used by browserify
   if (!opts.build) {
     require('./lib/string_pipe');
   }
+  if (!opts.test) {
```

Comment:
I'm missing the exact reason why this is can by disabled. Maybe add a comment what crashed during the build you mentioned?